packages/flutter/lib/src/cupertino/tab_view.dart

```diff
@@ -188,6 +188,7 @@ class _CupertinoTabViewState extends State<CupertinoTabView> {
         builder: routeBuilder,
         title: title,
         settings: settings,
+        useBarrierColor: false,
```

Comment:
I think this is going to remove the barrier during a tab-to-tab transition? And the barrier should be visible during that transition right?

Reply:
Yes it will... I didn't think about sub route trees in a tab going between tab views. Though it looks like the issue with the background is solved by using a `CupertinoPageScaffold` within `CupertinoTabView`. Should we point people towards that? Maybe close this PR and update the documentation? This is the implementation that the Cupertino codelab uses.